test/plugins/content/itemScrolling/test.js

```diff
@@ -72,8 +71,8 @@ define([
     ];
 
     QUnit.cases.init(pluginApi).test('plugin API ', function (data, assert) {
-        var runner = runnerFactory(providerName);
-        var plugin = pluginFactory(runner);
+        var testRunner = runnerFactory(providerName);
+        var plugin = pluginFactory(testRunner);
```

Comment:
This whole file was a bit confusing with `testRunner` and (item)`runner` defined in the top scope. I just did the minimum to make it pass.